src/modules/launcher/PowerLauncher/Helper/SingleInstance.cs

```diff
@@ -147,8 +148,34 @@ internal static class NativeMethods
 
 
         [DllImport("kernel32.dll", EntryPoint = "LocalFree", SetLastError = true)]
-        private static extern IntPtr _LocalFree(IntPtr hMem);
+        internal static extern IntPtr _LocalFree(IntPtr hMem);
 
+        [DllImport("user32.dll")]
+        internal static extern uint SendInput(uint nInputs, INPUT[] pInputs, int cbSize);
+
+        [DllImport("user32.dll", SetLastError = true)]
+        internal static extern int GetWindowLong(IntPtr hWnd, int nIndex);
+
+        [DllImport("user32.dll")]
+        internal static extern int SetWindowLong(IntPtr hWnd, int nIndex, int dwNewLong);
+
+        [DllImport("user32.dll")]
+        internal static extern IntPtr GetForegroundWindow();
+
+        [DllImport("user32.dll")]
+        internal static extern IntPtr GetDesktopWindow();
+
+        [DllImport("user32.dll")]
+        internal static extern IntPtr GetShellWindow();
+
+        [DllImport("user32.dll", SetLastError = true)]
+        internal static extern int GetWindowRect(IntPtr hwnd, out RECT rc);
+
+        [DllImport("user32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        internal static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
+
+        [DllImport("user32.DLL", CharSet = CharSet.Unicode)]
+        internal static extern IntPtr FindWindowEx(IntPtr hwndParent, IntPtr hwndChildAfter, string lpszClass, string lpszWindow);
```

Comment:
@somil55 Can you help me understand why these need to be added?  Where they moved from somewhere?

Reply:
They were moved from `WindowsInteropHelper.cs`. This was to fix [CA1060](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1060?view=vs-2019) error, which requires all native methods in a namespace to be moved to a static nativemethod class.